data/nixpkgs.json

```diff
@@ -0,0 +1,12 @@
+{
+  "branch": "release-21.05",
+  "description": "Nix Packages collection",
+  "homepage": "",
+  "owner": "NixOS",
+  "repo": "nixpkgs",
+  "rev": "5f244caea76105b63d826911b2a1563d33ff1cdc",
```

Comment:
Using that rev is outdated in the next days because the branch will advance with backports.

Reply:
That's fine. Half of the people will want to switch to a different channel like nixos-unstable anyways.

Reply:
@SuperSandro2000 you'll still be able to run `niv init --latest` or `niv init --nixpkgs NixOS/nixpkgs -b master`. Is that acceptable?

Reply:
Why do we need to add a nixpkgs Channel at all if it is totally unclear what people are going to use?

Reply:
Because most people just need "some" nixpkgs!

Reply:
I don't think anyone needs any random commit. Probably most if not all people either track master, unstable or release.

Reply:
I think this PR already improves the situation a fair bit by speeding up the initialization. Also it's fairly easy for users to pick nixpkgs master with `niv init --latest`, but I expect most people to just be happy with the stable release.

Reply:
Note: thanks for the feedback though! Don't hesitate to share more suggestions.